src/main/java/com/softserve/controller/AuthenticationController.java

```diff
@@ -53,14 +53,14 @@ public AuthenticationController(AuthenticationManager authenticationManager, Jwt
 
     @PostMapping("/sign-in")
     @ApiOperation(value = "Get credentials  for login")
-    public ResponseEntity signIn(@RequestBody AuthenticationRequestDTO requestDto) {
+    public ResponseEntity<Object> signIn(@RequestBody AuthenticationRequestDTO requestDto) {
```

Comment:
why not the actual return type? AuthenticationResponseDTO

Reply:
This is the situation with the previous comment.